maintainers/maintainer-list.nix

```diff
@@ -10268,6 +10268,12 @@
     githubId = 36407913;
     name = "Uli Baum";
   };
+  xfnw = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add xfnw``.

Reply:
done